app/controllers/teams_controller.rb

```diff
@@ -76,11 +76,31 @@ def delete_membership
     redirect_to @team
   end
 
+  def assign_membership_by_email
+    user = User.find_by_email params[:email]
+    if user
+      assign_membership_to_user user
+    else
+      flash[:error] = I18n.t('teams.invite_user_to_team.no_user_for_mail')
+    end
+    redirect_to @team
+  end
+
   def current_ability
     @current_ability ||= Ability.new(current_user, params[:team_member])
   end
 
   private
+    def assign_membership_to_user (new_member)
+      unless @team.members.include? new_member
```

Comment:
@bakoe I wasn't sure on which way of checking if a member is included we agreed on so I chose this simple approach. I could also replace this line by creating a new team_user association and checking if it is included in team.team_members...